backend/app/views/spree/admin/promotion_rules/create.js.erb

```diff
@@ -4,6 +4,12 @@ $('#rules .no-objects-found').hide();
 $('.product_picker').productAutocomplete();
 $('.user_picker').userAutocomplete();
 
+$('select.select2').select2({
```

Comment:
Ideally we wouldn't do this, but it is essentially the same as the `productAutocomplete`, `userAutocomplete` above.